<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>温度监测系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stack_view">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableWidget" name="tbl_sensors"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="act_global_view"/>
    <addaction name="act_boiler_view"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>历史查询</string>
    </property>
    <addaction name="act_query"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>系统设置</string>
    </property>
    <addaction name="act_change_pwd"/>
    <addaction name="act_account_man"/>
    <addaction name="act_moni_conf"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>数据备份/恢复</string>
    </property>
    <addaction name="act_backup"/>
    <addaction name="act_recover"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="doc_log">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>事件信息</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTableWidget" name="tbl_logs">
       <column>
        <property name="text">
         <string>时间</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>信息</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_global_view"/>
   <addaction name="act_boiler_view"/>
   <addaction name="act_moni_conf"/>
  </widget>
  <action name="act_global_view">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/images/global_view.png</normaloff>:/images/global_view.png</iconset>
   </property>
   <property name="text">
    <string>全局视图</string>
   </property>
  </action>
  <action name="act_boiler_view">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/images/boiler_view.gif</normaloff>:/images/boiler_view.gif</iconset>
   </property>
   <property name="text">
    <string>风炉视图</string>
   </property>
  </action>
  <action name="act_query">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>温度历史查询</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="act_change_pwd">
   <property name="text">
    <string>修改密码</string>
   </property>
  </action>
  <action name="act_account_man">
   <property name="text">
    <string>帐号管理</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="act_moni_conf">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>参数设置</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="act_backup">
   <property name="text">
    <string>数据备份</string>
   </property>
  </action>
  <action name="act_recover">
   <property name="text">
    <string>数据恢复</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
